### PR TITLE
feat: Support first-party Anthropic python SDK

### PR DESCRIPTION
**Pull request by anticorrelator**
Enables the use of Anthropic models (`claude-2.1`, `claude-instant-1.2`) with Phoenix evals.

Example:

```python

# ANTRHOPIC_API_KEY=##################
from phoenix.experimental.evals.models.anthropic import AnthropicModel

model = AnthropicModel()
model("hello! this is a test to see if you are working")

# > " Hello! Yes, I'm working. This is Claude, ready to assist."
```

### CODE REVIEW

File: src/phoenix/experimental/evals/models/anthropic.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+            self._tiktoken = tiktoken
+        except ImportError:
+            self._raise_import_error(
+                package_name="tiktoken",
+            )
+
+    def _init_client(self) -> None:
+        try:
+            import anthropic  # type:ignore
```
Comment (mikeldking) [inline src/phoenix/experimental/evals/models/anthropic.py:69]: let's add this to our `dev` deps and get type coverage

File: src/phoenix/experimental/evals/models/anthropic.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+            encoding = self._tiktoken.get_encoding("cl100k_base")
+        self._tiktoken_encoding = encoding
+
+    def _init_rate_limiter(self) -> None:
+        self._rate_limiter = RateLimiter(
+            rate_limit_error=self._anthropic.RateLimitError,
+            max_rate_limit_retries=10,
+            initial_per_second_request_rate=1,
+            maximum_per_second_request_rate=20,
+            enforcement_window_minutes=1,
```
Comment (mikeldking) [inline src/phoenix/experimental/evals/models/anthropic.py:93]: shouldn't these just be defaults?

**Review by mikeldking — APPROVED** [review]

File: src/phoenix/experimental/evals/models/anthropic.py (hunk 2 above)
Comment (anticorrelator) [reply src/phoenix/experimental/evals/models/anthropic.py:93]: yeah I think that is probably the right way to go. I am tempted to try to tune these per model based on our observed behavior of each API but perhaps it's too much given the variation between different users' rate limits

Comment (mikeldking) [reply src/phoenix/experimental/evals/models/anthropic.py:93]: Okay no worries - just trying to avoid magic numbers:)

**Pull request merged**